.props/Test.props

```diff
@@ -8,6 +8,22 @@
   <Import Project=".\_Common.props" />
   <Import Project=".\_AnalyzerSettings.props" />
 
+  <PropertyGroup>
+    <!-- Our test matrix includes every currently supported version of .NET
+          - net4.5.2 (EoL April 2022)
+          - net4.6.0 (EoL April 2022)
+          - net4.6.1 (EoL April 2022)
+          - net4.6.2
+          - net4.7.2
+          - net4.8.0
+          - netcoreapp3.1 (EoL Dec 2022)
+          - net5.0 (EoL Feb 2022)
+          - net6.0 (GA Nov 2021)
+    -->
+    <TargetFrameworks>net462;net472;net480;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) != 'Windows_NT'">netcoreapp3.1;net5.0</TargetFrameworks>
+  </PropertyGroup>
```

Comment:
FYI: although 452, 460, and 461 are still supported today, i don't see any value in onboarding any new projects to these frameworks.
Any test project that currently targets these can continue to target them like this:

`<TargetFrameworks>net452,net46,net461;$(TargetFrameworks)</TargetFrameworks>`

Reply:
Every test projects below has an entry for `TargetFrameworks`, why do we need to define this in `test.prop`